.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,7 @@
 <!-- This is a 🐛 bug fix. -->
 <!-- This is a 🙋 feature or enhancement. -->
 <!-- This is a 🔦 documentation change. -->
+<!-- This is a 🔨 refactoring. -->
```

Comment:
How about this which I think reads better
```suggestion
<!-- This is a 🔨 refactor. -->
```

Reply:
What do you think about `code refactoring`? 
Just like the phrase on wikipedia: https://en.wikipedia.org/wiki/Code_refactoring